bumble/avdtp.py

```diff
@@ -385,32 +408,37 @@ async def stop(self):
 
 
 # -----------------------------------------------------------------------------
-class MessageAssembler:  # pylint: disable=attribute-defined-outside-init
-    def __init__(self, callback):
-        self.callback = callback
```

Comment:
As a general design pattern, I'm not a big fan of using event emitters for direct 1:1 relationships like this, where there's normally just one client that wants to receive the data. Having a direct function allows better typing that generic events. Events are better suited for 1:M relationships, where you can dynamically add/remove listeners.

Reply:
Done